python/ray/_private/worker.py

```diff
@@ -2182,8 +2182,9 @@ def connect(
         # assumes that the directory structures on the machines in the clusters
         # are the same.
         # When using an interactive shell, there is no script directory.
+        # We also want to skip adding script directory when running from dashboard.
         code_paths = []
-        if not interactive_mode:
+        if not interactive_mode and not (namespace and namespace.endswith("dashboard")):
```

Comment:
where does this namespace get set? can we match the exist namespace instead of `endswith`?

Reply:
perfect